docs/src/css/custom.css

```
@@ -1,186 +1,105 @@
 /**
  * Any CSS included here will be global. The classic template
- * bundles Infima by default. Infima is a CSS framework designed to
- * work well for content-centric websites.
+ * bundles Infima by default.
  */
 
-/* You can override the default Infima variables here. */
+@import url('https://rsms.me/inter/inter.css');
```

Stylelint (17.7.0)

[error] Expected "url('https://rsms.me/inter/inter.css')" to be "'https://rsms.me/inter/inter.css'" (import-notation)

(import-notation)